ui/app/AppLayouts/Wallet/panels/BuyReceiveBanner.qml

```diff
@@ -0,0 +1,80 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import AppLayouts.Wallet.controls 1.0
+
+import StatusQ.Core.Theme 0.1
+
+Control {
+    id: root
+
+    property bool closeEnabled: true
+    property bool buyEnabled: true
+    property bool receiveEnabled: true
+
+    readonly property bool anyVisibleItems: buyCard.visible || receiveCard.visible
+
+    signal buyClicked()
+    signal receiveClicked()
+    signal closeBuy()
+    signal closeReceive()
+
+    contentItem: RowLayout {
+        id: layout
+        spacing: 16
+
+        BannerCard {
+            id: buyCard
+            objectName: "buyCard"
+            Layout.fillWidth: true
+            title: qsTr("Ways to buy crypto")
+            subTitle: qsTr("Via card or bank transfer")
+            image: Theme.png("wallet/wallet-green")
+            closeEnabled: root.closeEnabled
+            visible: implicitWidth > 0
+            implicitWidth: opacity > 0 ? layout.width / 2 : 0
+            opacity: root.buyEnabled ? 1 : 0
+
+            onClose: root.closeBuy()
+            onClicked: root.buyClicked()
+
+            Behavior on opacity {
+                NumberAnimation {
+                    duration: 250
+                }
+            }
+            Behavior on implicitWidth {
+                NumberAnimation {
+                    duration: 250
+                }
+            }
+        }
+        BannerCard {
+            id: receiveCard
+            objectName: "receiveCard"
+            Layout.fillWidth: true
+            title: qsTr("Receive crypto")
+            subTitle: qsTr("Deposit to your Wallet address")
+            image: Theme.png("wallet/flying-coin")
+            closeEnabled: root.closeEnabled
+            visible: implicitWidth > 0
+            implicitWidth: opacity > 0 ? layout.width / 2 : 0
+            opacity: root.receiveEnabled ? 1 : 0
+
+            onClose: root.closeReceive()
+            onClicked: root.receiveClicked()
+
+            Behavior on opacity {
```

Comment:
Also, this repeats here; what about folding it back to the BannerCard component, and make some proper state+transition?

Reply:
I don't see other reusable way to do it

Reply:
OK, let it be :)